apps/andi/lib/andi_web/controllers/edit_controller.ex

```diff
@@ -14,7 +14,7 @@ defmodule AndiWeb.EditController do
         |> render("404.html")
 
       dataset ->
-        if is_curator do
+        if is_curator and Andi.private_access?() do
```

Comment:
Not happy about all the mixing of concerns we have to do here, but the JSR is not there to change this dramatically